torch/fft/__init__.py

```diff
@@ -549,3 +555,177 @@
     tensor([ 2.0000+-0.0000j, -0.5000-0.6882j, -0.5000-0.1625j, -0.5000+0.1625j,
         -0.5000+0.6882j])
 """)
+
+fftfreq = _add_docstr(_fft.fft_fftfreq, r"""
+fftfreq(n, d=1.0, *, dtype=None, layout=torch.strided, device=None, requires_grad=False) -> Tensor
+
+Returns Discrete Fourier Transform sample frequencies for signal size :attr:`n`.
+
+Note:
+
+    By convention, the FFT returns positive frequency terms first, followed by
+    the negative frequencies in reverse order, so that ``f[-i]`` in python
```

Comment:
What is `i` referring to? 

"python" -> "Python"

Reply:
Clarified with "for all `i \leq n/2`".